src/components/editor/InputTarget.jsx

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Stanford University see Apache2.txt for license
+
+import React, {Component} from 'react';
+import {Typeahead} from 'react-bootstrap-typeahead'
+import PropTypes from 'prop-types'
+
+class InputTarget extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isLoading: true,
+      options: [],
+      selected: []
+    }
+  }
+
+  render() {
+    const target_url = './static/spoofedFilesFromServer/fromQA/frequencies.json'
```

Comment:
i think one of the points of the way I did spoofing is that it can get the URL it needs directly from the information in the resourceTemplate.   You wouldn't find "vocabulary:bf2:frequencies" in the resourceTemplate (which is how it's spoofed) -- you find the whole URL, no?   And something about that URL tells you to go to QA ... and THAT is what you need code for -- to translate what is in the resource template to getting the correct query to QA, with the help of generated javascript code from swagger client.

I think what we have here is actually an example of how we can use a Typeahead component ... which is something we need ... but is not the whole problem.  We need tickets for the rest of the work, no?

Reply:
👌 Nothing to change here right now? If I'm wrong, let's talk. I made ticket #226 and removed the QA spoof altogether. If we want a QA spoof that can be a separate ticket, but we should discuss whether we even need that fore the time being.